src/class-PluginUpdater.php

```diff
@@ -0,0 +1,248 @@
+<?php
+/**
+ * The PluginUpdater class which can be used to pull plugin updates from a new location.
+ * @package wpengine-geoip
+ */
+
+namespace WPEngine\Updates;
```

Comment:
❓ Is this namespace specific to this plugin?

> Please make sure you update the namespace in both from \YourPluginNamespace\Updates\PluginUpdater to something specific to your plugin. This will help us ensure no plugin uses an updater outside itself, and prevent issues with incompatible versions in the future.